applications/app/views/fragments/crosswords/crosswordMetaHeader.scala.html

```diff
@@ -35,14 +35,6 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(crosswordPage.i
                 </div>
             }
 
-            <div class="hide-from-leftcol crossword__clues-header">
-                <a href="https://app.adjust.com/16xt6hai" data-link-name="crossword-mobile-link">Download the Guardian app</a> for a better puzzles experience
-            </div>
-
-            <div class="hide-until-leftcol crossword__clues-header">
-                <a href="https://app.adjust.com/16xt6hai" data-link-name="crossword-desktop-link">Download the Guardian app</a> for a better puzzles experience
-            </div>
-
```

Comment:
Why are we removing the ad for the app?

Reply:
ah, great point, thanks!

I actually don't know how accessible the app versions are (I believe that Android and iOS have different players, too) but it seemed odd to me that we'd point users to the app from this page, unless we are specifically saying that the app is more accessible.

that said, strictly speaking it's outside the scope of this issue so I could just restore it to both pages, or have it toggled by `gridVisable` as you suggest above.

any thoughts on which is preferable? I don't know who 'owns' this link as such so not sure whether it's something I have discretion to remove or not..

Reply:
That sounds good to me. I've put the app ad back in, but only when `gridVisible`, as you suggested. I also took the opportunity to rename the variable while I was at it.

<img width="1481" alt="image" src="https://github.com/user-attachments/assets/b39baf3e-406d-4bfe-b107-85afb8d98700">
<img width="1488" alt="image" src="https://github.com/user-attachments/assets/1dacf34f-f4bf-4fbc-984a-ecb7b2f10bcc">

Reply:
Great work on this @bryophyta. I have added a small follow-up #27541 to hide the download app link on desktop.